dashboard/dashboard.conf

```diff
@@ -6,6 +6,7 @@ location ~ ^/pun/dev/dashboard(/.*|$) {
   passenger_enabled on;
 
   passenger_app_env development;
+  passenger_min_instances 1;
```

Comment:
Added an improvement to the Sid dashboard configuration so that the Sid is not shutdown when idle.